test/database.ts

```diff
@@ -840,9 +840,12 @@ describe('Database', () => {
       try {
         database.writeAtLeastOnce(null, (err, res) => {});
       } catch (err) {
-        (err as grpc.ServiceError).message.includes(
-          "Cannot read properties of null (reading 'proto')"
-        );
+        // Performing a substring search on the error because
+        // depending on the version of Node.js, the error might be either of:
+        //  * Cannot read properties of null (reading 'proto')
+        //  * Cannot read property 'proto' of null
+        (err as grpc.ServiceError).message.includes('Cannot read propert');
```

Comment:
instead of this do something like 
const errorMessage = (err as grpc.ServiceError).message;
errorMessage.includes("Cannot read properties of null (reading 'proto')") || 
  errorMessage.includes("Cannot read property 'proto' of null")

Reply:
Awesome and thank you @surbhigarg92!